content/io/cloudslang/base/utils/noop.sl

```
@@ -0,0 +1,22 @@
+#   (c) Copyright 2015-2017 Hewlett-Packard Enterprise Development Company, L.P.
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+########################################################################################################################
+#!!
+#! @description: noop should do "nothing". It simulates this behavior by calling the `pass` Python statement.
```

builder should fail if `@description` is missing - right now it doesn't - @tethryus WDYT?